package.json

```
@@ -27,7 +27,7 @@
     "nodemailer": "^6.3.1",
     "open": "^6.4.0",
     "pify": "^4.0.1",
-    "pug": "^2.0.4",
+    "pug": "3.0.0",
```

```suggestion
    "pug": "^3.0.0",
```
else we need to release an update every-time pug releases one.

---

cool makes sense and done :)

---

thanks for reviewing, do i have your blessing to remove node 8 from `.travis.yml`?

---

there is no other options. we have to remove it.

cc @niftylettuce